com.microsoft.mrtk.input/Interactors/InteractorVisuals/MRTKLineVisual.cs

```diff
@@ -70,12 +70,12 @@ public Gradient SelectActiveColorGradient
         }
 
         [SerializeField]
-        [Tooltip("Controls the length of the visual that the gradient is applied relative to the ray interactor's max raycast distance")]
+        [Tooltip("On hit, the gradient will be applied evenly along the line renderer until it's total length is longer than this value multiplied by the ray interactor's max raycast distance")]
         [Range(0.01f, 1)]
-        float maxGradientLength = 0.1f;
+        float maxGradientLength = 0.3f;
```

Comment:
Could you explain a bit more about changing the default value?

Reply:
The previous value was a little too aggressive and would end up compressing the gradient in situations where it didn't really improve usability. Increasing the value makes it so we don't unnecessarily apply the compression effect (saving us on performance)